src/hpke/mod.rs

```diff
@@ -148,8 +148,9 @@ mod tests {
     }
 
     impl<R: RngCore + CryptoRng> EncryptionSuite<R> {
+        const MKP_ORIGIN: &'static str = "";
         const HELPER_ORIGIN: &'static str = "foo";
-        const SITE_ORIGIN: &'static str = "bar";
+        const SRD: &'static str = "xn--mozilla.com.xn--example.com";
```

Comment:
"xn--" hah

Reply:
this was my interpretation of this sentence

```
The ASCII serialization of the registrable domain for the current site as a period-separated sequence of
A-labels
```

by changing it I wanted to validate my understanding by you - it seems we have to prefix everything with `xn--` according to the [spec](https://www.rfc-editor.org/rfc/rfc5890#section-2.3.2.5) ?

Reply:
Ah, my error: NR-LDH is also OK.  These are perfectly OK as NR-LDH labels.